packages/shared/styles/components/atoms/SfCheckbox.scss

```
@@ -7,51 +7,62 @@
     width: var(--checkbox-container-width);
     cursor: var(--checkbox-cursor, pointer);
   }
+
   &__checkmark {
     box-sizing: border-box;
     display: flex;
     align-items: center;
     justify-content: center;
-    width: var(--checkbox-width, var(--checkbox-size, 1.25rem));
-    height: var(--checkbox-height, var(--checkbox-size, 1.25rem));
-    border: var(--checkbox-border, 1px solid var(--c-gray-variant));
+    width: var(--checkbox-width, var(--checkbox-size, 1.5rem));
+    height: var(--checkbox-height, var(--checkbox-size, 1.5rem));
```

```suggestion
    width: var(--checkbox-size, 1.5rem);
    height: var(--checkbox-size, 1.5rem);
```
in this case we can use just, because it should be always square, wdyt?

---

indeed, I agree :)